CryptoPkg/Test/UnitTest/Library/BaseCryptLib/AeadAesGcmTests.c

```diff
@@ -52,6 +52,10 @@ TestVerifyAeadAesGcm (
   UINT8    OutTag[1024];
   UINTN    OutTagSize;
 
+  if (!PcdGetBool (PcdCryptoServiceAeadAesGcmEncrypt) || !PcdGetBool (PcdCryptoServiceAeadAesGcmDecrypt)) {

```

Comment:
Do these not need MU_CHANGE tags?

Reply:
You're right that they should be included.  I've updated the PR.